modules/reactor/src/positions/HexagonalGridPositions.C

```diff
@@ -58,9 +58,10 @@ HexagonalGridPositions::HexagonalGridPositions(const InputParameters & parameter
         std::set<unsigned int>(getParam<std::vector<unsigned int>>("include_in_pattern").begin(),
                                getParam<std::vector<unsigned int>>("include_in_pattern").end()))
 {
-  if (MooseUtils::absoluteFuzzyGreaterThan(_nr * _pin_pitch * sqrt(3), _lattice_flat_to_flat))
+  if (MooseUtils::absoluteFuzzyGreaterThan((_nr - 1) * _pin_pitch * sqrt(3), _lattice_flat_to_flat))
```

Comment:
so if you have 1 ring, then lattice_flat_to_flat should be > 0 ?
it seems 1 * sqrt(3) * pin_pitch worked better

Reply:
That is a good point. Though now thinking about it, maybe 1 ring is more like a special case? Maybe we check for `_nr==1` (where the flat to flat has to be at least the pin pitch). This doesn't seem to follow the formula for when `nr>1`. (i.e. when nr=0, there isn't a sqrt(3) factor needed).

Reply:
I dont think so?
let s just rederive the formula

Reply:
I'm not quite sure I know what you mean. I guess it also depends on if you want it to work for `nr` as integers only. E.g. the example in the issue counting as 6 or 5.5.

Let's say `nr=1`, how would you define the minimum flat to flat?

Reply:
I tried with the logic we discussed (separate `nr=1`), but ran into this error
```
*** ERROR ***
Specified bundle pitch 0.069282 is too small to fit the given pins and wire wrap!
```
from `HexagonalLatticieUtils.C`.

What exactly are `wire_diameter` and `wire_pitch`? They show up here
```
HexagonalLatticeUtils::HexagonalLatticeUtils(const Real bundle_inner_flat_to_flat,
                                             const Real pin_pitch,
                                             const Real pin_diameter,
                                             const Real wire_diameter,
                                             const Real wire_pitch,
                                             const unsigned int n_rings,
                                             const unsigned int axis)
```
They have fixed values of `0.` and `1.` inside `HexagonalGridPositions::initialize()`